interface/forms/newGroupEncounter/common.php

```diff
@@ -334,7 +334,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
      <td class='bold' nowrap><?php echo xlt('Additional Date:'); ?></td>
      <td class='text' nowrap><!-- default is blank so that while generating claim the date is blank. -->
       <input type='text' size='10' class='datepicker' name='form_onset_date' id='form_onset_date'
-       value='<?php echo $viewmode && $result['onset_date']!='0000-00-00 00:00:00' ? substr($result['onset_date'], 0, 10) : ''; ?>'
+       value='<?php echo $viewmode && $result['onset_date']!='0000-00-00 00:00:00' ? oeFormatShortDate(substr($result['onset_date'], 0, 10)) : ''; ?>'
```

Comment:
The 0000-00-00 00:00:00 stuff here seems really odd. Does this every get outputted?